Yoast/yoast-components
# Upgrade eslint-yoast-config

## Summary

This PR can be summarized in the following changelog entry:

* _N/A_

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* See the `travis` build complete successfully.

Fixes #730
